.gitignore

```diff
@@ -131,4 +131,5 @@ dmypy.json
 .idea/
 
 .docker/
-.dump.rdb
\ No newline at end of file
+.dump.rdb
+dump.rdb
```

Comment:
Just as a small reference for the future: https://github.com/gridsingularity/d3a-api-client/pull/120#discussion_r598592011

Reply:
(If you use pyCharm this can be done automatically: `Editor > General > Ensure  every saved file ends with a line break`)